serverless.yml

```diff
@@ -13,7 +13,30 @@ functions:
       DB_USER: ${opt:DB_USER}
       DB_PASSWORD: ${opt:DB_PASSWORD}
     events:
-      - http: 
+      - http:
           path: projects
           method: get
-          cors: true
\ No newline at end of file
+          cors: true
```

Comment:
Can you also update serverless.yml to match master (https://github.com/GrokItTeam/grokit-backend/blob/master/serverless.yml) plus the two events you have added